.github/release.yml

```diff
@@ -0,0 +1,21 @@
+# The GitHub release configuration file: https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#configuring-automatically-generated-release-notes
```

Comment:
This action would never be used, as the raspbian image is based on the apt-pkg.   Am thinking it should be nuked.

Reply:
If I remember correctly this would only happen when a draft release is generated.